packages/hash/integration/src/tests/model/knowledge/user.model.test.ts

```diff
@@ -92,6 +95,35 @@ describe("User model class", () => {
 
     expect(fetchedUser).not.toBeNull();
 
-    expect(fetchedUser).toEqual(createdUser);
+    expect(fetchedUser?.entityId).toEqual(createdUser?.entityId);
```

Comment:
Why has this been changed

Reply:
See https://github.com/hashintel/hash/pull/1068#discussion_r972872788